packages/terra-button-group/docs/README.md

```diff
@@ -19,12 +21,85 @@ import ButtonGroup from 'terra-button-group';
     <ButtonGroup.Button text="Button 1" key="button1" />
     <ButtonGroup.Button text="Button 2" key="button2" />
   </ButtonGroup>
+```
 
-// Selectable (toggle style) button group with second button pre-selected
-  <ButtonGroup isSelectable>
-    <ButtonGroup.Button text="Button 1" key="button1" />
-    <ButtonGroup.Button isSelected text="Button 2" key="button2" />
-  </ButtonGroup>
+### Single-Select Button Group
+```jsx
+import React from 'react';
+import ButtonGroup from '../../src/ButtonGroup';
+
+class SingleSelectButtonGroup extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { selectedKey: ['1'] };
```

Comment:
Because selectedKey is singular this should probably be a single key that just gets put into an array
```
this.state = { selectedKey: '1' };
```
```
<ButtonGroup onChange={this.handleSelection} selectedKeys={[this.state.selectedKey]}>
```

Reply:
Good catch: https://github.com/cerner/terra-core/pull/1155/commits/c4e0000d6ceaf96b85053f0046154de1799e7072